examples/modules/cfd/pages/spring/README.adoc

```diff
@@ -0,0 +1,71 @@
+# Spring CFD
+:page-vtkjs: true
+## Description of the example
+
+In this example, we simulate a fluid evolving in a spring mesh using the Stokes equation.
+
+## Geometry
+
+image:spring_cfd/springmesh.png[50%]
+The mesh can be retrieved from girder with the following ID: 5b03dfcbb0e957402704806c (see xref:contribute:girder:README.adoc[girder]).
```

Comment:
why not provide a link here?

Reply:
Trying to be coherent with accepted examples, such as http://docs.feelpp.org/examples/0.105/thermoelectric/electromagnet/README/

Reply:
I already mentioned to @romainhild that I did not understand why we have the girder ID.